### ISSUE TITLE
[Feature Request] Enabled Admins to modify servers added/created by other users

### ISSUE THREAD

**Issue by BloodyIron** [splewis/get5-web#151]
If I set myself and another STEAMID as admins, and that other user creates/adds a server to the ecosystem, only they can modify that server configuration (but other people can add them if they are public).

This is a problem because I myself, being an admin, cannot login as their STEAM account to modify it.

I would like to see a way whereby anyone listed as an Admin can modify any/all Servers that are added, whether they are public or not.

How's that sound? :)

**Comment by TandelK** [comment]
Basically the system defined in this eco system is that the Main Admin that might be you Create the Server  and allow them to be Public Servers. After that the Other Admins can use that server while creating matches. The same server will also be visible to Whitelisted IDs for match creation.

**Comment by BloodyIron** [comment]
Yeah but if we don't want the servers to be public, but we need multiple admins to modify (IP changes, otherwise) servers that were created by any number of different admins, today that's not possible for one admin to modify servers created by another admin. This is what I'm referring to.

If you want servers dedicated to a tournament, and only that, and you have multiple admins, this becomes problematic.
